config/settings/development.yml

```diff
@@ -21,3 +21,5 @@ wasapi_providers:
         accountid: 1
         username: user
         password: pass
+crawl_directory: tmp/jobs
+wasapi_downloader_bin: /Users/jlittman/data/was/wasapi-downloader/build/install/wasapi-downloader/bin/wasapi-downloader
```

Comment:
I think you do NOT want your laptop dir here ...  maybe even "/add/local/dir/to/wasapi-downloader" or whatever.

Reply:
Ooops. Forgot that.